core/org.eclipse.cdt.core/plugin.xml

```diff
@@ -72,6 +72,16 @@
          </run>
       </cextension>
    </extension>
+   <extension
+         id="GNU_PE64"
```

Comment:
With the addition of this new parser, I suppose the [documentation](https://github.com/eclipse-cdt/cdt/blob/main/build/org.eclipse.cdt.managedbuilder.core/schema/ManagedBuildTools.exsd#L517) needs to be updated accordingly.

Reply:
Duplicate comment.